data/services/local-registry-ocp/start-local-registry.service

```
@@ -0,0 +1,20 @@
+[Unit]
+Description=Local Registry (OCP)
+Wants=network.target
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=/etc/assisted/registry.env
+ExecStartPre=/bin/rm -f %t/%n.ctr-id
+ExecStartPre=/usr/local/bin/setup-local-registry.sh
+ExecStart=/usr/bin/podman run --net host --cidfile=%t/%n.ctr-id --privileged --replace --log-driver=journald --name=registry -p 5000:5000 -p 5443:5000 -v ${REGISTRY_DATA}:/var/lib/registry -v /tmp/certs:/certs -e REGISTRY_HTTP_ADDR=0.0.0.0:5000 -e REGISTRY_HTTP_TLS_CERTIFICATE=certs/domain.crt -e REGISTRY_HTTP_TLS_KEY=certs/domain.key -u 0 --entrypoint=/usr/bin/distribution ${REGISTRY_IMAGE} serve /etc/registry/config.yaml
```

```suggestion
ExecStart=/usr/bin/podman run --net host --cidfile=%t/%n.ctr-id --privileged --replace --log-driver=journald --name=registry -v ${REGISTRY_DATA}:/var/lib/registry -v /etc/iri-registry/certs:/certs:ro -e REGISTRY_HTTP_ADDR=0.0.0.0:22625 -e REGISTRY_HTTP_TLS_CERTIFICATE=/certs/tls.crt -e REGISTRY_HTTP_TLS_KEY=/certs/tls.key -u 0 --entrypoint=/usr/bin/distribution ${REGISTRY_IMAGE} serve /etc/registry/config.yaml
```

In order to better support pivoting the appliance to MCO I'll suggest to keep a similar format:
* Use port 22625
* Use /etc/iri-registry/certs as certs folder
* Use tls.key/tls.cert as cert filenames

Note: 22625 is required